docker-entrypoint.sh

```diff
@@ -17,7 +17,7 @@ _ip_address() {
 	# scrape the first non-localhost IP address of the container
 	# in Swarm Mode, we often get two IPs -- the container IP, and the (shared) VIP, and the container IP should always be first
 	ip address | awk '
-		$1 == "inet" && $2 !~ /^127[.]/ {
+		$1 == "inet" && $2 !~ /^127[.]/ && $NF != "lo" {
```

Comment:
Do you think it's worth dropping the `$2 !~ /^127[.]/` comparison too since checking for `lo` is more accurate?

Reply:
Done.